### PR TITLE
Feature/hw 53099 core get transfer

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: Sources/Hyperwallet.swift [hunk 1]
```diff
@@ -305,6 +305,24 @@ public final class Hyperwallet {
...
+    ///
+    /// The `completion: @escaping (HyperwalletTransfer?, HyperwalletErrorType?) -> Void` that is passed in to this
+    /// method invocation will receive the successful response(HyperwalletTransfer) or error(HyperwalletErrorType)
+    /// from processing the request.
+    ///
+    /// - Parameters:
+    ///   - transferMethodToken: the Hyperwallet specific unique identifier for the `HyperwalletTransfer`
+    ///                          being requested
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+    public func getTransfer(transferMethodToken: String,
```
Comment (syyang-hw) [inline Sources/Hyperwallet.swift]: should be transferToken not transferMethodToken

File: Sources/Hyperwallet.swift [hunk 2]
```diff
@@ -305,6 +305,24 @@ public final class Hyperwallet {
...
+                            completion: @escaping (HyperwalletTransfer?, HyperwalletErrorType?) -> Void) {
+        httpTransaction.performRest(httpMethod: .get,
+                                    urlPath: "transfers/\(transferMethodToken)",
```
Comment (syyang-hw) [inline Sources/Hyperwallet.swift]: transferToken

File: Sources/Hyperwallet.swift [hunk 3]
```diff
@@ -305,6 +305,24 @@ public final class Hyperwallet {
...
     }
 
+    /// Returns the `HyperwalletTransfer` linked to the transfer method token specified, or nil if none exists.
+    ///
+    /// The `completion: @escaping (HyperwalletTransfer?, HyperwalletErrorType?) -> Void` that is passed in to this
+    /// method invocation will receive the successful response(HyperwalletTransfer) or error(HyperwalletErrorType)
+    /// from processing the request.
+    ///
+    /// - Parameters:
+    ///   - transferMethodToken: the Hyperwallet specific unique identifier for the `HyperwalletTransfer`
```
Comment (syyang-hw) [inline Sources/Hyperwallet.swift]: transferToken

File: Tests/Transfer/HyperwalletTransfer.swift [hunk 4]
```diff
@@ -85,10 +85,37 @@ class HyperwalletTransferTests: XCTestCase {
         XCTAssertEqual(error.getHttpCode(), 400)
         XCTAssertEqual(error.getHyperwalletErrors()?.errorList?.first?.code, "INVALID_DESTINATION_TOKEN")
     }
+
+    func testGetPayPalAccount_success() {
```
Comment (syyang-hw) [inline Tests/Transfer/HyperwalletTransfer.swift]: why this name?

File: Tests/Transfer/HyperwalletTransfer.swift [hunk 5]
```diff
@@ -85,10 +85,37 @@ class HyperwalletTransferTests: XCTestCase {
...
+        let response = HyperwalletTestHelper.okHTTPResponse(for: "CreateTransferResponse")
+        let url = String(format: "%@/transfers/trf-123456", HyperwalletTestHelper.restURL)
+        let request = HyperwalletTestHelper.buildGetRequest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var transferResponse: HyperwalletTransfer?
+        var errorResponse: HyperwalletErrorType?
+
+        // When
+        Hyperwallet.shared.getTransfer(transferMethodToken: "trf-123456", completion: { (result, error) in
```
Comment (syyang-hw) [inline Tests/Transfer/HyperwalletTransfer.swift]: transferToken

File: Sources/Hyperwallet.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Hyperwallet.swift]: Renamed

File: Sources/Hyperwallet.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Hyperwallet.swift]: Renamed

File: Sources/Hyperwallet.swift (hunk 3 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Hyperwallet.swift]: Renamed

File: Tests/Transfer/HyperwalletTransfer.swift (hunk 4 above)
Comment (oleksii-lubianyi-epam) [reply Tests/Transfer/HyperwalletTransfer.swift]: Have no clue. Renamed

File: Tests/Transfer/HyperwalletTransfer.swift (hunk 5 above)
Comment (oleksii-lubianyi-epam) [reply Tests/Transfer/HyperwalletTransfer.swift]: Renamed

**Comment by oleksii-lubianyi-epam** [comment]
Created branch with proper name. New PR to be created

**Pull request closed without merging**